portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path

```diff
@@ -88,6 +88,11 @@
 	<td>//label[contains(.,'${key_checkboxName}')]/input[@type='checkbox' and contains(@class,'disabled')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CHECKBOX_DISABLED_EXPORT_TRANSLATION</td>
```

Comment:
Can we refactor this path and CHECKBOX_DISABLED and remove "CHECKBOX"? That's already mentioned in the test file. So the locator should be Checkbox#DISABLED_EXPORT_TRANSLATION

Reply:
see https://github.com/austinchiang/liferay-portal/pull/831